.github/workflows/tests.yaml

```diff
@@ -39,6 +40,8 @@ jobs:
         php-version: ${{ matrix.php }}
         coverage: none # disable xdebug, pcov
     - run: composer install --no-suggest --no-progress
+    - run: composer update --no-suggest --no-progress
```

Comment:
that will make the tests very very slow. what is your intention?

Reply:
If someone installs surf as (global) composer package, the dependencies will be installed without respecting the lock only by resolving deplendecies by the semver definitions of the json. Therefore we should test it, to.

Reply:
I'm against such checks. That makes only sense before a new release but not for every PR.

Reply:
As a release can be installed at any time with dependencies of that secound, another idea is to run this check scheduled.

Reply:
As the scheduled run is only possible for the default branch, 
> Scheduled workflows run on the latest commit on the default or base branch
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onschedule
and the update only adds 10 secounds, I don't see a time problem.

Reply:
what exactly could be the problem when you install surf with the latest dependencies?

Reply:
See last problems of TYPO3 CMS with updated dependency https://forge.typo3.org/issues/91832. Sometimes minor updates ship unexpected breaking changes.
Some also run tests with `--prefer-lowest` to check if the minimum required version with all dependencies is ok. See test matrix of TYPO3 CMS https://github.com/TYPO3/TYPO3.CMS/blob/master/Build/bamboo/src/main/java/core/AbstractCoreSpec.java#L1711-L1766.
They run test against the locked (install), latest update ald lowest dependencies.

Reply:
After all:

- It doesn't hurt.
- We had the tests with travis, too https://github.com/TYPO3/Surf/blob/2.0/.travis.yml#L62-L73